src/code.cloudfoundry.org/garden-external-networker/cni/cni_loader_test.go

```diff
@@ -62,21 +62,6 @@ var _ = Describe("GetNetworkConfig", func() {
 		})
 	})
 
-	Context("when a valid config file exists", func() {
```

Comment:
Can you keep a test with a conf file that ends in ".conf" and test what happens to it now?